### ISSUE TITLE
move to python3, django 2.0

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#68]
https://docs.djangoproject.com/en/2.0/releases/2.0/

**Comment by sc0Vu** [comment]
@owocki, I can work on this.

**Comment by owocki** [comment]
@sc0Vu how complex do u think it is

**Comment by speizerj** [comment]
Hey all - I really love what Gitcoin is doing and I'd like to help as well - I think porting to python 3 is a sizable undertaking but definitely worth it.

I have the project running locally using the Docker image, and was able to convert said docker image to 3.6 and Django 2.0. The app runs successfully though I didn't test out too many pages to check for library issues.

I'm happy to work through some part of the app if you want to divide and conquer, @sc0Vu as I imagine there are a lot of quick fixes just with syntax stuff (exceptions, etc)

Or if there's something easier that I can get started with, I'm game for that as well.

**Comment by owocki** [comment]
> I have the project running locally using the Docker image, and was able to convert said docker image to 3.6 and Django 2.0. 

awesome!  want to PR the main repo with your work so far?  from there you and @sc0Vu can divide/conquer (up to you guys)

**Comment by speizerj** [comment]
absolutely, just created pr #70 

@sc0Vu what do you think the best approach is for dividing the work? I can get the repo to a point where the local server is running without issue, and then we can divide from there? hard to debug otherwise

**Comment by sc0Vu** [comment]
@speizerj the docker file looks good and I think it hard to divide the work, maybe you can do this by your own :)
Please tell me if there is anything I can help!

**Comment by owocki** [comment]
thx for working it out amonst yourselves.  @sc0Vu if you want to check https://gitcoin.co/explorer for another issue you think might be a fix, now is a good time!  hit me up on slack if you can't find one

**Comment by mbeacom** [comment]
If you need any help with this, feel free to ping me. I recently migrated a few of my own internal projects from 2.7|1.11 -> 3.6|2.0.

**Comment by speizerj** [comment]
Thanks Mark, much appreciated. Working on it today - if I run into any
snags I'll reach out.

**Comment by amites** [comment]
How's this effort going? Been working on similar tasks for other projects of recent and happy to contribute

**Comment by speizerj** [comment]
@amites check out #70 

we're just waiting on testing all the flows but i think it's very close
